sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/CallToAsyncMethodShouldNotBeBlocking.cs

```diff
@@ -36,6 +36,8 @@ public sealed class CallToAsyncMethodShouldNotBeBlocking : SonarDiagnosticAnalyz
     {
         private const string DiagnosticId = "S4462";
         private const string MessageFormat = "Replace this use of '{0}' with '{1}'.";
+        private const string ResultName = "Result";
```

Comment:
It would be nice to use constant for `SleepName` as well. It's also used in code separately.